sandy/core/samples.py

```diff
@@ -132,60 +132,65 @@ def test_shapiro(self, size=None, pdf="normal"):
 
         Examples
         --------
-        Generate 5000 xs samples normally and log-normally distributed
+        Generate 5000 xs samples normally, log-normally and uniform distributed
         >>> tape = sandy.get_endf6_file("jeff_33", "xs", 10010)
         >>> njoy_kws = dict(err=1, errorr33_kws=dict(mt=102))
         >>> nsmp = 5000
-        >>> smp_norm = tape.get_perturbations(nsmp, njoy_kws=njoy_kws, smp_kws=dict(seed33=1, pdf="normal"))[33]
-        >>> smp_lognorm = tape.get_perturbations(nsmp, njoy_kws=njoy_kws, smp_kws=dict(seed33=1, pdf="lognormal"))[33]
-
-        The Shapiro-Wilks test proves right for the normal samples and the normal distribution.
-        >>> stat_norm = []
-        >>> stat_lognorm = []
-        >>> for nsmp in [10, 50, 100, 500, 1000, 5000]:
-        ...    df = smp_norm.test_shapiro(pdf="normal", size=nsmp)
-        ...    idx = df.statistic.idxmin()
-        ...    stat_norm.append(df.loc[idx].rename(nsmp))
-        ...
-        ...    df = smp_norm.test_shapiro(pdf="lognormal", size=nsmp)
-        ...    idx = df.statistic.idxmin()
-        ...    stat_lognorm.append(df.loc[idx].rename(nsmp))
+        >>> seed = 5
         >>>
-        >>> opts = dict(left_index=True, right_index=True, suffixes=("_norm", "_lognorm"))
-        >>> df = pd.DataFrame(stat_norm).merge(pd.DataFrame(stat_lognorm), **opts).rename_axis("# SMP")
-        >>> print(df)
-               statistic_norm  pvalue_norm  statistic_lognorm  pvalue_lognorm
-        # SMP                                                                
-        10        9.19610e-01  3.15408e-01        9.40333e-01     6.43714e-41
-        50        9.44342e-01  1.84070e-02        9.40333e-01     6.43714e-41
-        100       9.78776e-01  1.03195e-01        9.40333e-01     6.43714e-41
-        500       9.85038e-01  5.02533e-05        9.40333e-01     6.43714e-41
-        1000      9.89657e-01  1.68679e-06        9.40333e-01     6.43714e-41
-        5000      9.90618e-01  1.03590e-17        9.40333e-01     6.43714e-41
+        >>> smp_norm = tape.get_perturbations(nsmp, njoy_kws=njoy_kws, smp_kws=dict(seed33=seed, pdf="normal"))[33]
+        >>> smp_lognorm = tape.get_perturbations(nsmp, njoy_kws=njoy_kws, smp_kws=dict(seed33=seed, pdf="lognormal"))[33]
+        >>> smp_uniform = tape.get_perturbations(nsmp, njoy_kws=njoy_kws, smp_kws=dict(seed33=seed, pdf="uniform"))[33]
+
+        In this example we defined the following arbitrary convergence criteria:
+            - if the p value is larger than 0.05 we fail to reject the null-hypothesis and we accept the results
```

Comment:
Should a reference for the 0.05 criterion be mentioned? To me it seems to be a quite agreed one. I mean, it's a bit more than an "arbitrary convergence criterion", isn't it?

Reply:
Not really needed, it is not used by sandy. It is just an example!